python/tk_multi_workfiles/work_area.py

```diff
@@ -190,89 +203,117 @@ def resolve_user_sandboxes(self):
+             - Publish Area: <Sgtk TemplatePath shot_publish_area_maya: sequences/{Sequence}/{Shot}/{Step}/publish/maya>
+             - Publish: <Sgtk TemplatePath maya_shot_publish: sequences/{Sequence}/{Shot}/{Step}/publish/maya/{name}.v{version}.{extension}>
+
+        :returns The formatted string.
+        """
+        return ("CTX: %s\n - Work Area: %s\n - Work: %s\n - Publish Area: %s\n - Publish: %s"
+                % (self._context,
+                   self.work_area_template, self.work_template,
                    self.publish_area_template, self.publish_template)
                 )
 
     def _load_settings(self):
         """
+        Extracts the settings from the environment file.
         """
         # attemps to load all settings for the context:
-        templates_to_find = ["template_work", "template_publish", 
+        templates_to_find = ["template_work", "template_publish",
                              "template_work_area", "template_publish_area"]
-        settings_to_find = ["saveas_default_name", "saveas_prefer_version_up", 
+        settings_to_find = ["saveas_default_name", "saveas_prefer_version_up",
                             "version_compare_ignore_fields", "file_extensions"]
         resolved_settings = {}
-        app = sgtk.platform.current_bundle()
-        try:
-            if self._context:
-                resolved_settings = self._get_settings_for_context(self._context, templates_to_find, settings_to_find)
-        except:
-            app.log_exception("There was an error parsing the settings for context '%s'" % self._context)
-        finally:
-            # update the templates and settings regardless if an exception was raised.
-            #
-
-            # update templates:
-            self.work_area_template = resolved_settings.get("template_work_area")
-            self.work_template = resolved_settings.get("template_work")
-            self.publish_area_template = resolved_settings.get("template_publish_area")
-            self.publish_template = resolved_settings.get("template_publish")
-
-            # update other settings:
-            self.save_as_default_name = resolved_settings.get("saveas_default_name", "")
-            self.save_as_prefer_version_up = resolved_settings.get("saveas_prefer_version_up", False)
-            self.version_compare_ignore_fields = resolved_settings.get("version_compare_ignore_fields", [])
-            extensions = resolved_settings.get("file_extensions") or []
-            extensions = [ext if ext.startswith(".") else ".%s" % ext for ext in extensions if ext]
-            self.valid_file_extensions = extensions
-
-            # test for user sandboxes:
-            self._work_template_contains_user = self.work_template and bool(self._get_template_user_keys(self.work_template))
-            self._publish_template_contains_user = self.publish_template and bool(self._get_template_user_keys(self.publish_template))
+        if self._context:
```

Comment:
Of all this block, only the if self._context block is interesting. The rest is jus an indentation change. I removed the try catch so exceptions were not swallowed by the except clause and instead reported back to the caller.